addons/vehicles/CfgVehicles/Viper.inc

```diff
@@ -0,0 +1,191 @@
+#define CAMO_NET_TEXTURE QUOTE(A3\Armor_F\Data\camonet_AAF_Digi_Green_CO.paa)
+
+class SWS_V_MBT_M700_Viper : I_MBT_03_cannon_F {
+    ITEM_META(2);
+    displayName = QNAME(M700 Viper);
+    faction = "SWS_UNSC";
+    side = 1;
+    crew = "OPTRE_UNSC_Marine_Soldier_Crewman";
+    typicalCargo[] = { "OPTRE_UNSC_Marine_Soldier" };
+    editorPreview = QPATHTOF(data\viper\editor_preview.jpg);
+    
+    radarType=2;
```

Comment:
Spacing should be consistent on property definitions; there's no linter yet but it hurts my eyes. It'll need to be fixed throughout here.

```suggestion
    radarType = 2;
```